alignak/property.py

```diff
@@ -198,8 +198,8 @@ def __init__(self, text=None):
         self.unused = True
 
 
-_boolean_states = {'1': True, 'yes': True, 'true': True, 'on': True,
-                   '0': False, 'no': False, 'false': False, 'off': False}
+__boolean_states__ = {'1': True, 'yes': True, 'true': True, 'on': True,
```

Comment:
Seems better to define this in class BoolProp() because it is used only in this class